product_docs/docs/pgd/5/reference/pgd-settings.mdx

```diff
@@ -478,19 +478,21 @@ Track lock timing when tracking statistics for relations.
 
 ### `bdr.enable_wal_decoder`
 
-Enables Logical Change Record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this is false. When set to true, a decoding worker process starts and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.
+Enables logical change record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this setting is false. When set to true, a decoding worker process starts, and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.
 
-Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.
+!!! Note
+    You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to true on the group.
 
 ### `bdr.receive_lcr`
 
-Enables the node to request Logical Change Records are used when subscribed to another node. By default, this is false. When this setting is true on a downstream node, the node requests that upstream nodes use Logical Change Records (LCR) when sending to it.  If `bdr.enable_wal_decoder` is set to true on a node, this setting should also be `true`.
+Enables the node to request to use LCRs when subscribed to another node. By default, this setting is false. When this setting is true on a downstream node, the node requests that upstream nodes use LCRs when sending to it. If you set `bdr.enable_wal_decoder` to true on a node, also set this setting to `true`.
 
-Note: This setting also needs to be enabled on all nodes in the PGD group and for the `enable_wal_decoder` option to be set to true on the group.
+!!! Note 
+    You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to to true on the group.
```

Comment:
```suggestion
 You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to to true on the group.
!!!
```

Fence rather than indent admonitions - it avoids triggering an admonition bug and is much more clearly defined.